src/libs/actions/Policy.js

```diff
@@ -166,6 +166,16 @@ function isPolicyOwner(policy) {
     return _.keys(loginList).includes(policy.owner);
 }
 
+/**
+ * Is the user the member of the given policy?
+ *
+ * @param {Object} policy
+ * @returns {Boolean}
+ */
+function isPolicyMember(policy) {
```

Comment:
We don't need this, we already have a policy admin util

https://github.com/Expensify/App/blob/ed21fdaaf90f458f7a063eaeb346bff61bac133e/src/libs/PolicyUtils.js#L101